deploy/kubernetes/setup.yaml

```diff
@@ -1,21 +1,21 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: csi-packet-standard
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
-provisioner: csi.packet.net
+provisioner: net.packet.csi
 parameters:
   plan: standard
 volumeBindingMode: Immediate
 
 ---
 
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: csi-packet-performance
-provisioner: csi.packet.net
+provisioner: net.packet.csi
```

Comment:
I'm curious why this was reverted from https://github.com/packethost/csi-packet/issues/87